test/postgresql.order.test.js

```diff
@@ -0,0 +1,224 @@
+// Copyright IBM Corp. 2013,2019. All Rights Reserved.
+// Node module: loopback-connector-postgresql
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+'use strict';
+
+require('./init');
+const should = require('should');
+const async = require('async');
+
+let db, PostWithDefaultSort, PostWithDisabledDefaultSort, PostWithNumericDefaultSort, PostWithNumericStringDefaultSort;
+
+describe('Order settings', function() {
+  before(function() {
+    db = global.getSchema();
+  });
+
+  before(function(done) {
+    PostWithDefaultSort = db.define(
+      'PostWithDefaultSort',
+      {
+        id: {type: Number, id: true},
+        title: {type: String, length: 255, index: true},
+        content: {type: String},
+      },
+      {
+        defaultSort: true,
+      },
+    );
+
+    PostWithDisabledDefaultSort = db.define(
+      'PostWithDisabledDefaultSort',
+      {
+        id: {type: Number, id: true},
+        title: {type: String, length: 255, index: true},
+        content: {type: String},
+      },
+      {
+        defaultSort: false,
+      },
+    );
+
+    PostWithNumericDefaultSort = db.define(
+      'PostWithNumericDefaultSort',
+      {
+        id: {type: Number, id: true},
+        title: {type: String, length: 255, index: true},
+        content: {type: String},
+      },
+      {
+        defaultSort: 'numericIdOnly',
+      },
+    );
+
+    PostWithNumericStringDefaultSort = db.define(
+      'PostWithNumericStringDefaultSort',
+      {
+        id: {type: String, id: true},
```

Comment:
Just wondering, would it be possible to have a database with mixed id types (e.g. both `String` and `Number`) and what would happen in that case when it's sorted?

Reply:
Hi @nabdelgadir , 
Good question! It is possible, though not very common AFAIK. 
By default it would be sorted, but once set to `numericIdOnly` it doesn't sort them. This is because the implementation checks if all ID types are `Number` ID types. If they aren't all `Number` types (for instance, one is a `String`), it won't sort.

Another way to go would be to only sort the ID's that _are_ a `Number` type, and skip any other type, but I think that might be messy.

Reply:
Thank you for the explanation!